aspnetcore/host-and-deploy/linux-nginx.md

```diff
@@ -92,14 +92,22 @@ If no [ForwardedHeadersOptions](/dotnet/api/microsoft.aspnetcore.builder.forward
 
 ### Install Nginx
 
+Use `apt-get` to install Nginx. The installer creates a System V init script that runs Nginx as daemon on system startup. 
```

Comment:
System V? Doesn't 16.04 use systemd?

Reply:
Idk ... It was a pre-existing remark in the topic ... I just moved it up.

Reply:
Seems correct ... https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units ...

> Please note that although systemd has become the default init system for many Linux distributions ...

... they go on to say that some distros don't use it. They say to just run `systemctl` ... if it claims it isn't installed, then some other init system is in place. Of course, I'll find out when I run the steps.